File: frontend/src/app/api/models/specified-time-dto.ts
/* tslint:disable */
import { UserBaseInfoDTO } from './user-base-info-dto';

export interface SpecifiedTimeDTO {

  endTime?: string;

  id?: number;

  repeatInterval?: string;

  startTime?: string;

  user?: UserBaseInfoDTO;

  duration?: number;

  groupId?: number;
}

File: backend/interview_portal/src/main/java/by/interview/portal/dto/SpecifiedTimeDTO.java
package by.interview.portal.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.time.Period;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SpecifiedTimeDTO {

    private Long id;

    private LocalDateTime startTime;

    private LocalDateTime endTime;

    private Period repeatInterval;

    private UserBaseInfoDTO user;

    private Integer duration;

    private Long groupId;
}
